app/scripts/modules/core/instance/details/multipleInstances.controller.js

```diff
@@ -56,6 +56,22 @@ module.exports = angular.module('spinnaker.core.instance.details.multipleInstanc
       });
     };
 
+    this.canTerminateInstancesAndShrinkServerGroups = () => {
+      return !this.selectedGroups.some((group) => {
+        // terminateInstancesAndShrinkServerGroups is aws-only
+        return group.cloudProvider !== 'aws';
```

Comment:
The optimized terminateAndShrink is aws-only ... not sure if there's a better strategy to avoid the slippery slope of certain actions being cloud provider specific.

For example, the `reboot` operation isn't supported by `titus` (and perhaps other providers) but isn't gated by anything.